.github/workflows/wheels.yml

```diff
@@ -0,0 +1,34 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - main
+  pull_request
+
+jobs:
+
+  build-wheel:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        py_ver: [ "3.7", "3.8", "3.9" ]
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.py_ver }}
+          architecture: x64
+      - name: Checkout functorch
+        uses: actions/checkout@v2
+      - name: Install PyTorch Nightly
+        run: |
+          python3 -mpip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+      - name: Build wheel
+        run: |
+          python3 -mpip install wheel
+          python3 setup.py bdist_wheel
+      - name: Upload wheel as GHA artifact
+        uses: actions/upload-artifact@v2
```

Comment:
@malfet where does this end up uploading to?

Reply:
To projects actions page, for example https://github.com/pytorch/functorch/actions/runs/1604493500